packages/libxinerama.rb

```diff
@@ -22,8 +22,9 @@ class Libxinerama < Package
      x86_64: '73f25bf7cf51787d00e2fbe46cf6589e321c465333e8d2c9dc3d97bfa93a70da',
   })
 
-  depends_on 'fontconfig'
+  depends_on 'libx11'
   depends_on 'libxcb'
+  depends_on 'libxext'
```

Comment:
Does this affect the binaries?  If not then we don't need to bump the version.

Reply:
I don't think this affects the binaries at all. This is a build issue. Those were already being pulled in from xorg_libs in any case, so the binaries would already know about them.

Reply:
Might as well update to https://www.x.org/archive/individual/lib/libXinerama-1.1.4.tar.bz2.

Reply:
Done.